.github/workflows/daily_ci.yml

```diff
@@ -0,0 +1,12 @@
+# This workflow runs every weekday at 15:00 UTC (8AM PDT)
```

Comment:
This is fine and an improvement over the current state, but any reason not to include the other CI?

Reply:
currently, this is the only job that is triggered daily, we could add the other ones; except osx - since you need to physically click run for those. we could add the others, I just wanted to get back to what we had